ci/build_win32.bat

```diff
@@ -39,10 +40,26 @@ cd build-win32
 @rem build pxScene
 cmake -DCMAKE_VERBOSE_MAKEFILE=ON ..
 cmake --build . --config Release -- /m
+if %errorlevel% neq 0 exit /b %errorlevel%
+
 cpack .
+if %errorlevel% neq 0 exit /b %errorlevel%
 
 @rem create standalone archive
 cd _CPack_Packages/win32/NSIS
 7z a -y pxscene-setup.zip pxscene-setup
 
 cd %ORIG_DIR%
+
+@rem deploy artifacts
+@rem based on: https://www.appveyor.com/docs/build-worker-api/#push-artifact
+if "%APPVEYOR%"=="True" (
+    if "%DEPLOY_ARTIFACTS%"=="True" (
+        @rem NSIS based installer
+        appveyor PushArtifact "build-win32\\_CPack_Packages\\win32\\NSIS\\pxscene-setup.exe" -DeploymentName "installer" -Type "Auto" -Verbosity "Normal"

```

Comment:
Looks like we are already inside NSIS folder in line 49. So providing the path as "build-win32\\_CPack_Packages\\win32\\NSIS\\pxscene-setup.exe" might fail. The artifact name alone should work fine.

Kindly clarify.

Reply:
>Looks like we are already inside NSIS folder in line 49. So providing the path as "build-win32\_CPack_Packages\win32\NSIS\pxscene-setup.exe" might fail. 

It was implemented according to documentation: https://www.appveyor.com/docs/build-worker-api/#push-artifact
> `<path>`: the path to the file to upload, relative to your project’s checkout